css-route-matching/index.bs

```diff
@@ -122,42 +128,61 @@ as follows:
 	: from: <<route-location>>
 	:: The result is true if
 		the [=document's navigation API=] of the document
-		is non-null,
-		its [=transition=] is non-null,
-		its [=from entry=]'s {{NavigationHistoryEntry/url}}
-		is non-null and
-		[=URL pattern/match|matches=]
-		the [=route location URL pattern=] of <<route-location>>.
+		is non-null, and either:
+
+		* its [=transition=] is non-null,
+			its [=from entry=]'s {{NavigationHistoryEntry/url}}
+			is non-null and
+			[=URL pattern/match|matches=]
+			the [=route location URL pattern=] of <<route-location>>.
+
+		* its [=activation=] is non-null,
```

Comment:
Not exactly. It should remain valid for the duration of the task (or "update the rendering steps") in which [=reveal a document=] has been called. It's probably ok to put it here in a handwavy way and add an issue to put this in the HTML spec.